.github/workflows/benchmark.yml

```diff
@@ -105,9 +105,11 @@ jobs:
         run: cd tpcds-kit/tools && make OS=LINUX
       - name: Install Java ${{ github.event.inputs.jdk }}
         if: steps.cache-tpcds-sf-1.outputs.cache-hit != 'true'
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v3
         with:
+          distribution: 'zulu'
           java-version: ${{ github.event.inputs.jdk }}
+          check-latest: true
```

Comment:
Is there any specifc reason to use check-latest?

This might cause use latest java without cache, then build might slower (more time to download latest java).

https://github.com/actions/setup-java#check-latest

Reply:
> Is there any specifc reason to use check-latest?
> 
> This might cause use latest java without cache, then build might slower (more time to download latest java).
> 
> https://github.com/actions/setup-java#check-latest

Ok, I will close this one,  for `check-latest: true` because I have not finished reading the document, so it is temporarily set to true